### PR TITLE
Tweaks and minor fixes

### PR DESCRIPTION
**Pull request by DannyS03**
- knocks off a few pending fixes
- ~outdated links~ EDIT: will open in follow up PR

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ In addition to the implementations above, the libp2p community is actively worki
 
 ## Community
 
-Get in touch with other members of the libp2p community who are building tools and applications with libp2p! You can ask questions, discuss new ideas, or get support for problems at https://discuss.ipfs.io, but you can also [hop on IRC](/community/irc/) for a quick chat.
+Get in touch with other members of the libp2p community who are building tools and applications with libp2p! You can ask questions, discuss new ideas, or get support for problems at [IPFS discussion forums](https://discuss.ipfs.io), but you can also [hop on IRC](irc://irc.freenode.org/%23libp2p) for a quick chat.
```
Comment (marten-seemann) [inline content/_index.md:38]: Should this link to the libp2p forums?

File: content/introduction/_index.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ In addition to the implementations above, the libp2p community is actively worki
 
 ## Community
 
-Get in touch with other members of the libp2p community who are building tools and applications with libp2p! You can ask questions, discuss new ideas, or get support for problems at https://discuss.ipfs.io, but you can also [hop on IRC](/community/irc/) for a quick chat.
+Get in touch with other members of the libp2p community who are building tools and applications with libp2p! You can ask questions, discuss new ideas, or get support for problems at [IPFS discussion forums](https://discuss.ipfs.io), but you can also [hop on IRC](/community/irc/) for a quick chat.
```
Comment (marten-seemann) [inline content/introduction/_index.md:39]: here too?

File: content/introduction/what-is-libp2p.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ In a world with billions of networked devices, knowing who you're talking to is
...
 
-libp2p supports "upgrading" a connection provided by a [transport](#transport) into a securely encrypted channel. The process is flexible, and can support multiple methods of encrypting communication. libp2p currently supports [TLS 1.3](https://www.ietf.org/blog/tls13/) and [Noise](https://noiseprotocol.org/), though not every language implementation of libp2p supports both of these. (Older versions of libp2p may support a [deprecated](https://blog.ipfs.io/2020-08-07-deprecating-secio/) protocol called SECIO; all projects should switch to TLS 1.3 or Noise instead.)
+libp2p supports "upgrading" a connection provided by a [transport](#transport) into a securely encrypted channel. The process is flexible, and can support multiple methods of encrypting communication. libp2p currently supports [TLS 1.3](https://www.ietf.org/blog/tls13/) and [Noise](https://noiseprotocol.org/), though not every language implementation of libp2p supports both of these. (Older versions of libp2p may support a [deprecated](https://blog.ipfs.io/2020-08-07-deprecating-secio/) protocol; all projects should switch to TLS 1.3 or Noise instead.)
```
Comment (marten-seemann) [inline content/introduction/what-is-libp2p.md:44]:
```suggestion
libp2p supports "upgrading" a connection provided by a [transport](#transport) into a securely encrypted channel. The process is flexible, and can support multiple methods of encrypting communication. libp2p currently supports [TLS 1.3](https://www.ietf.org/blog/tls13/) and [Noise](https://noiseprotocol.org/), though not every language implementation of libp2p supports both of these. (Older versions of libp2p may support the [deprecated secio protocol](https://blog.ipfs.io/2020-08-07-deprecating-secio/); all projects should switch to TLS 1.3 or Noise instead.)
```

File: content/_index.md (hunk 1 above)
Comment (DannyS03) [reply content/_index.md]: updated

File: content/introduction/_index.md (hunk 2 above)
Comment (DannyS03) [reply content/introduction/_index.md]: updated

**Review by marten-seemann — APPROVED** [review]

**Pull request merged**